packages/vuepress-theme-odyssey/styles/base/_fonts.scss

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@font-face {
+  font-family: 'Whyte';
+  font-style: normal;
+  font-weight: normal;
+  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-Book.woff') format('woff');
```

Comment:
@jmelberg-okta should we worry about exposing this in an Okta OSS repo? This package isn't published to NPM, but the code is available to all.

Reply:
moved to #728